### DISCUSSION TITLE
3D Effect for Corners of the Image ?

### DISCUSSION THREAD

**Discussion by SheepDomination** [ImageMagick/ImageMagick#6665] [General]
I would like to take an image and apply a 3D effect to the corners of the image; as in a square or a triangle with control over it's size and how much of the image is applies as a 3D effect ?

**Comment by fmw42** [comment]
Your request is vague.  Please explain what you mean by "3D effect" and/or post an example of what you want. If you can create it in Photoshop, explain the steps and perhaps we can repeat it in Imagemagick.
